portway/portway
# Form error/success/submitting tracking

* Tracks which form is doing what 
* Displays a success icon if the form was submitted successfully
* Displays an error message if the form had errors
* Displays a spinner if the form is submitting
* Adds max image size on avatars

**To test**

I commented out some client-side validation for testing purposes

1. Go to User Profile
2. Submit the "Your information" form, deleting the user's name (bad data)
3. Submit the avatar form with a successful image
4. You should see errors for the first one, success for the second one

Rinse repeat for http://localhost:3000/d/admin/organization
